relayer/strategies.go

```diff
@@ -119,6 +121,33 @@ func FilterOpenChannels(channels []*types.IdentifiedChannel, openChannels []*Act
 	return openChannels
 }
 
+// ApplyChannelFilterRule will use the given ChannelFilter's rule and channel list to build the appropriate list of
+// channels to relay on.
+func ApplyChannelFilterRule(filter *ChannelFilter, channels []*types.IdentifiedChannel) []*types.IdentifiedChannel {
```

Comment:
This function is exported -- not sure whether it needs to be. 

Either way, let's get some tests to confirm the filtering behavior.

Reply:
I've got a half baked test similar to our other Docker tests so I'll get that finished up and also add some basic tests making sure the new functions/methods behave as expected! 👍